syntaxes/test/cases.json

```diff
@@ -14,6 +14,12 @@
     "grammarFiles": ["syntaxes/inline-styles.json"],
     "testFile": "syntaxes/test/data/inline-styles.ts"
   },
+  {
+    "name": "expression syntax",
+    "scopeName": "expression.ng",
+    "grammarFiles": ["syntaxes/out/expression.json"],
```

Comment:
The tests look a little off right now, because they all point to `expression.ng`, but do not show any of the new scopes you have created in the expression grammar. You will probably have to delete the dummy grammar and add `syntaxes/template.json` to the grammar files here.